File: app/src/main/java/com/driveawayz/SignUp/signupphone/response/SignUp1User.java
package com.driveawayz.SignUp.signupphone.response;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class SignUp1User {

@SerializedName("access_token")
@Expose
private String accessToken;

public String getAccessToken() {
return accessToken;
}

public void setAccessToken(String accessToken) {
this.accessToken = accessToken;
}

}

File: app/src/main/java/com/driveawayz/SignUp/signupphone/response/SignUp1user.kt
package com.driveawayz.SignUp.signupphone.response

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class SignUp1user {

    @SerializedName("access_token")
    @Expose
    private var accessToken: String? = null

    fun getAccessToken(): String? {
        return accessToken
    }

    fun setAccessToken(accessToken: String?) {
        this.accessToken = accessToken
    }

}